src/Products/Product.php

```diff
@@ -87,10 +88,11 @@ private function set_brand() {
 	 */
 	private function set_category() {
 
-		$return        = '';
-		$taxonomy_name = ShoppingFeedHelper::wc_category_taxonomy();
+		$return           = '';
+		$taxonomy_name    = ShoppingFeedHelper::wc_category_taxonomy();
+		$sf_yoast_options = get_option( Admin\Options::SF_YOAST_OPTIONS );
```

Comment:
We should be using `ShoppingFeedHelper::get_sf_yoast_options()` here instead of loading the option value directly.

Reply:
I made the change